usaon_vta_survey/templates/survey.html

```diff
@@ -3,7 +3,8 @@
 
 {% block content %}
 
-  <h2>{% block title %}Survey: {{survey.id}}{% endblock %}</h1>
+  <h2>{% block title %}Survey: {{survey.id}}{% endblock %}</h2>
+  <h3>Survey: {{survey.title}}</h3>
```

Comment:
Maybe the title should be in the larger header and the id in the smaller header? Doesn't really matter, just a thought :)